Bluetooth: Mesh: Add ext adv to samples (#8513)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Added overlay file with support for storing data with :ref:`emds_readme`.
    Also changed the sample to restore Light state after power-down.

> * All Bluetooth mesh samples are now using the extended advertiser as default advertiser, instead of legacy advertiser.

Reviewer: ```suggestion
* Default advertiser changed to extended advertiser instead of legacy advertiser for all Bluetooth mesh samples.
```
